configuration/configuration.go

```diff
@@ -132,7 +154,7 @@ func (c *ConfigurationData) setConfigDefaults() {
 	//---------
 	c.v.SetTypeByDefaultValue(true)
 	c.v.SetDefault(varPostgresHost, "localhost")
-	c.v.SetDefault(varPostgresPort, 5433)
+	c.v.SetDefault(varPostgresPort, 5432)
```

Comment:
It looks like you just copied the configuration from WIT and overrode the existing Auth config. They are not the same! :)